docs/logical_data_model.md

```diff
@@ -57,17 +57,72 @@ class RequestErrors {
   * updatedAt : timestamp
 }
 
+class Role {
+  * id : integer
+  * name : string
+}
+
+class Topic {
+  * id : integer
+  * name : string
+}
+
+class RoleTopic {
+  * id : integer <<generated>>
+  * roleId : integer(32) REFERENCES public.Roles.id
+  * topicId: integer(32) REFERENCES public.Topics.id
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Goal {
+  * id : integer
+  * name : string
+  status : string
+  timeframe : string
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class TopicGoal {
+  * id : integer
+  * roleId : integer(32) REFERENCES public.Roles.id
+  * topicId: integer(32) REFERENCES public.Topics.id
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Grantee {
+  * id : integer
+  * name : string
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Ttaplan {
+  * id : integer <<generated>>
+  * granteeId : integer(32) REFERENCES public.Grantees.id
```

Comment:
Is the plan specific to a grantee (by name) or a grant number?  I'm asking because as a result of our pairing yesterday Josh and I were thinking we'd have two tables for grant information. We'd have `grant` (includes grant `number`) with a many to one relationship to `grantee` (includes `name` of organization receiving the grant).

This is a visual in case that helps.
<img width="934" alt="Screen Shot 2020-12-02 at 1 13 19 PM" src="https://user-images.githubusercontent.com/9893700/100920335-94281a80-34a0-11eb-84fa-bb19c6282692.png">

Reply:
> Is the plan specific to a grantee (by name) or a grant number?

The plan is specific to a grant number with an assumption that one grantee would have multiple grants listed in the "Grantee (distinct by row)" column. More info about the columns was added to the import script. Hope that helps.